### PR TITLE
fix: edited design for PhoneNumberTunnel on cart page 

### PR DESCRIPTION
**Pull request by deepak1-1**
## Description
Updated design for phone number tunnel on cart page while placing order

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Changes and Fixes
- Description of changes and all fixes in this issue
## Screenshots 
image

(prefer all screen sizes images)

## Checklist
- [ ] Database schema is updated
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Looks good on large screens
- [ ] Looks good on mobiles
- [ ] Looks good on Tablets

Affected Apps
- [x] Store
    - [ ] SEO
    - [ ] Menu
    - [ ] Checkout
    - [ ] Add to Cart logic
    - [ ] Location Logic
    - [ ] Product page
    - [ ] Login
    - [ ] Payment
    - [x] Kiosk
- [ ] Admin
    - [ ] Order
    - [ ] CRM
    - [ ] Product app
    - [ ] Analytics
    - [ ] Settings
    - [ ] Brand
    - [ ] Config Builder
    - [ ] Coupons
    - [ ] Locations
    - [ ] Users
    - [ ] Inventory
- [ ] Server
    - [ ] Payment
    - [ ] Email
    - [ ] SMS
    - [ ] Notifications
    - [ ] Integration
    - [ ] Hasura auth
- [ ] Template Engine

### CODE REVIEW

File: store/src/components/kiosk/component/PhoneNumberTunnel.js [hunk 1]
```diff
@@ -246,60 +248,204 @@ export const PhoneNoWarningModal = ({
...
    setVisible,
    callback,
-   config
+   config,
+   triggeredFrom
 }) => {
    const { t } = useTranslation()
    const { currentPage } = useConfig()
+   console.log("triggeredFrom: ", triggeredFrom)
+
```
Comment (FAsami) [inline store/src/components/kiosk/component/PhoneNumberTunnel.js:257]: Please remove the unnecessary console.log()
```suggestion
```

File: store/src/components/kiosk/component/PhoneNumberTunnel.js [hunk 2]
```diff
@@ -246,60 +248,204 @@ export const PhoneNoWarningModal = ({
...
+               </button>
+            </div>
+         </>
+         }
+         { (triggeredFrom === "cartPage") && 
+            <>
+               <div className='ant-modal-title'>
+                  <svg width="481" 
+                        height="300" 
+                        viewBox="0 0 481 300"
```
Comment (FAsami) [inline store/src/components/kiosk/component/PhoneNumberTunnel.js:316]: Move  this icon/illustration  to the assets folder or config based  use a PNG image

**Comment by deepak1-1** [comment]
test it

**Review by FAsami — APPROVED** [review]

**Comment by deepak1-1** [comment]
@FAsami please do test this branch

File: store/src/components/kiosk/fulfillment.js [hunk 3]
```diff
@@ -246,6 +246,46 @@ export const FulfillmentSection = props => {
                   </div>
                )}
          </div>
+         {!kioskAvailability['ONDEMAND_PICKUP'] &&
```
Comment (FAsami) [inline store/src/components/kiosk/fulfillment.js:249]: We don't require this as we are using different screen for it

**Comment by deepak1-1** [comment]
@FAsami Okay I create a new PR with new branch. It is having many changes that I haven't made!

**Comment by deepak1-1** [comment]
@FAsami test it!

**Pull request merged**